.github/scripts/hermetic_library_generation.sh

```diff
@@ -99,22 +94,18 @@ docker run \
   -v "$(pwd):${workspace_name}" \
   -v "$HOME"/.m2:/home/.m2 \
   -e GENERATOR_VERSION="${generator_version}" \
-  gcr.io/cloud-devrel-public-resources/java-library-generation:"${image_tag}" \
-  --baseline-generation-config-path="${workspace_name}/${baseline_generation_config}" \
-  --current-generation-config-path="${workspace_name}/${generation_config}"
+  gcr.io/cloud-devrel-public-resources/java-library-generation:"${image_tag}"
 
 # commit the change to the pull request.
-rm -rdf output googleapis "${baseline_generation_config}"
+rm -rdf output googleapis
 git add --all -- ':!pr_description.txt'
 changed_files=$(git diff --cached --name-only)
 if [[ "${changed_files}" == "" ]]; then
-    echo "There is no generated code change with the generation config and Dockerfile change ${config_diff}."
```

Comment:
We actually mentioned `Dockerfile change` in the message but it was not achieved before?

Reply:
Dockerfile is not part of generation config, so the previous `docker run` won't generate anything if Dockerfile is the only change.